werf-giterminism.yaml

```
@@ -0,0 +1,6 @@
+giterminismConfigVersion: 1
+config:
+  dockerfile:
+    allowContextAddFiles:
+      - packages/sdk-playground/dist
+      - packages/sdk-playground/.env
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)